hass_nabucasa/remote.py

```diff
@@ -299,15 +306,26 @@ async def connect(self) -> None:
             _LOGGER.error("Connection problem to snitun server")
         except RemoteBackendError:
             _LOGGER.error("Can't refresh the snitun token")
+        except RemoteInsecureVersion:
+            self.cloud.client.user_message(
+                "connect_remote_insecure",
+                "Home Assistant Cloud error",
+                "Can't initialize connection because of insecure instance marked",
```

Comment:
```suggestion
                "Remote connection is disabled because this Home Assistant instance is marked as insecure. For more information and to enable it again, visit the [Nabu Casa Account page](https://account.nabucasa.com).",
```